app/src/main/java/org/mozilla/tv/firefox/IntentValidator.kt

```diff
@@ -65,7 +65,7 @@ object IntentValidator {
                 if (TextUtils.isEmpty(dataString)) {
                     return null // We can't create a session from an Intent without a URL.
                 }
-
+                TelemetryIntegration.INSTANCE.viewIntentEvent()
```

Comment:
I feel like we should be logging all intents (rather than just View ones), along with what their type is.  

The story specifies that YouTube fling events should be logged, for example, and if I understand correctly those are `ACTION_MAIN` intents.

Reply:
We already log YouTube fling events. What other type is there besides view?

Reply:
> We already log YouTube fling events.

I'm blind.  Thanks.

> What other type is there besides view?

I don't know if there are any now, I'd rather log all intents rather than risk missing an update that adds a new one.  This way feels a little brittle.  Then again, we don't want to update the YouTube fling event on short notice, and we don't want to double up on them either.  So I suppose we should leave it like this for now.